src/Meshes/abstractMesh.ts

```diff
@@ -350,6 +350,8 @@ export class AbstractMesh extends TransformNode implements IDisposable, ICullabl
      */
     public isPickable = true;
 
+    public isNearInteractable = false;
```

Comment:
nit: I think isNearPickable is more aligned with the other predicate boolean, isPickable.